lib/scraping/hpi_data_collector.rb

```diff
@@ -91,7 +91,7 @@ def get_html_document(url)
     document = ''
 
     begin
-      document = Nokogiri::HTML(URI.open(@base_url + url, allow_redirections: :all))
+      document = Nokogiri::HTML(URI.parse(@base_url + url).open(allow_redirections: :all))
```

Comment:
This change breaks the scraper, as well as the tests.

Reply:
Not sure why, I ran them afterwards and they were all passing for me